### DISCUSSION TITLE
CUSA04529 - Uncharted 4 A Thief's End [1.32]

### DISCUSSION THREAD

**Discussion by ichopster** [GoldHEN/GoldHEN_Cheat_Repository#5] [Cheat requests]
Please add CUSA04529 - Uncharted 4 A Thief's End [1.32]

**Comment by kmeps4** [comment]
Will check if any available..

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
